assets/css/custom.css

```diff
@@ -424,7 +424,7 @@ body:last-child .btn-u, x:-moz-any-link {
 .browse-courses {
     display: inline-block;
     border-radius: 6px;
-    margin: 20px 0px 0px 10px;
+    margin: 20px 10px 12px;
```

Comment:
It's worth trying implement vertical alignment with table and table-cell display values. It's more stable, e.g. changing the font will break the vertical alignment. Example:
html:
<div class="parent">
  <div class="child">Content here</div>
</div>
	
css:
.parent {display: table}

.child {
  display: table-cell;
  vertical-align: middle;
 }